### ISSUE TITLE
Lively wallaper's UI is black

### ISSUE THREAD

**Issue by Nadorn** [rocksdanister/lively#1369]
**Describe the bug**
Lively wallpaper is still working in the taskbar but when i open Lively wallaper's UI  the UI turn black.

**To Reproduce**
I don't know

**Expected behavior**
Make it work again

**Screenshots/Video**
If applicable, add screenshots/video to help explain your problem.
![image](https://user-images.githubusercontent.com/112544016/187628068-3e5b87e2-8684-4829-87b8-426edaf61d74.png)
![image](https://user-images.githubusercontent.com/112544016/187628386-ee1ccf90-e212-4e22-82a9-583e2f7f4393.png)
![image](https://user-images.githubusercontent.com/112544016/187628160-d4d979ad-0ce9-4443-bc59-bd50d86fd882.png)

**Comment by rocksdanister** [comment]
1. What's your Windows 10 version? (Startmenu -> winver)
2. Check `Windows Event Viewer` logs (Windows Logs -> Application)  for `Lively.UI.WinUI.exe` error.
3. Share the application logfile(s):
 `C:\Users\<UserName>\AppData\Local\Lively Wallpaper\logs`
or if Microsoft store version of Lively -
`%LocalAppData%\Packages\12030rocksdanister.LivelyWallpaper_97hta09mmv6hy\LocalCache\Local\Lively Wallpaper\logs`

**Comment by Nadorn** [comment]
> 1. What's your Windows 10 version? (Startmenu -> winver)
I'm using Windows 11 right now.
> 2. Check `Windows Event Viewer` logs (Windows Logs -> Application)  for `Lively.UI.WinUI.exe` error.
> 3. Share the application logfile(s):
[20220831_145206.txt](https://github.com/rocksdanister/lively/files/9460146/20220831_145206.txt)
[20220831_151321.txt](https://github.com/rocksdanister/lively/files/9460147/20220831_151321.txt)
[20220831_151337.txt](https://github.com/rocksdanister/lively/files/9460148/20220831_151337.txt)
[20220831_155819.txt](https://github.com/rocksdanister/lively/files/9460149/20220831_155819.txt)

**Comment by rocksdanister** [comment]
>Check Windows Event Viewer logs (Windows Logs -> Application) for Lively.UI.WinUI.exe error.

**Comment by Nadorn** [comment]
> > Check Windows Event Viewer logs (Windows Logs -> Application) for Lively.UI.WinUI.exe error.

![image](https://user-images.githubusercontent.com/112544016/187664718-6a5b8dc2-3d21-4ea3-adb4-f73ed6e1b97e.png)

**Comment by apec000** [comment]
@Nadorn I have this problem too, did you fix it?

**Comment by Nadorn** [comment]
> @Nadorn I have this problem too, did you fix it?

no i didn't know how to fix it.

**Comment by rocksdanister** [comment]
Try disable these software if you have any running:
`H24, MSI Afterburner, RivaTuner Statistics Server, HW64`

**Comment by Nadorn** [comment]
> 
I think i didn't see any of these running
do i need to disable in the task manager right? or if there any other way to disable can you tell me?

**Comment by risingtop** [comment]
I am also got the same problem on Windows 11 with Lively Wallpaper from Microsoft Store,
I hope my log might help you investigate more on the issue.

[20220916_224716.txt](https://github.com/rocksdanister/lively/files/9593948/20220916_224716.txt)
[20220917_102806.txt](https://github.com/rocksdanister/lively/files/9593949/20220917_102806.txt)
[20220917_153519.txt](https://github.com/rocksdanister/lively/files/9593950/20220917_153519.txt)
[20220918_095501.txt](https://github.com/rocksdanister/lively/files/9593951/20220918_095501.txt)
[20220916_204127.txt](https://github.com/rocksdanister/lively/files/9593952/20220916_204127.txt)

**Comment by rocksdanister** [comment]
Found a similar issue for PowerToys https://github.com/microsoft/PowerToys/issues/20502

**Comment by crutkas** [comment]
due to license, i can't quickly glance at source.  Is the application a WinAppSDK?

**Comment by rocksdanister** [comment]
@crutkas Yes, UI is WinUI 3 (store release is packaged, installer is unpackaged.)

**Comment by rocksdanister** [comment]
Can you try changing region format https://github.com/rocksdanister/lively/issues/2187#issuecomment-2049024898
